docs/source/en/api/pipelines/text_to_video.md

```diff
@@ -18,11 +18,11 @@ specific language governing permissions and limitations under the License.
 
 # Text-to-video
 
-[VideoFusion: Decomposed Diffusion Models for High-Quality Video Generation](https://huggingface.co/papers/2303.08320) is by Zhengxiong Luo, Dayou Chen, Yingya Zhang, Yan Huang, Liang Wang, Yujun Shen, Deli Zhao, Jingren Zhou, Tieniu Tan.
+[ModelScope Text-to-Video Technical Report](https://arxiv.org/abs/2308.06571) is by Jiuniu Wang*, Hangjie Yuan*, Dayou Chen*, Yingya Zhang*, Xiang Wang, Shiwei Zhang.
```

Comment:
```suggestion
[ModelScope Text-to-Video Technical Report](https://arxiv.org/abs/2308.06571) is by Jiuniu Wang, Hangjie Yuan, Dayou Chen, Yingya Zhang, Xiang Wang, Shiwei Zhang.
```

Reply:
Would you mind removing the `*`? I don't think we need to italicize every other author :)

Reply:
@stevhliu  I add them to denote equal contribution. But, you can remove it if it is more appropriate.